digital_asset_types/src/dao/mod.rs

```diff
@@ -245,6 +249,26 @@ impl SearchAssetsQuery {
             joins.push(rel);
         }
 
+        if let Some(n) = self.name.to_owned() {
+            let name_as_str = std::str::from_utf8(&n).map_err(|_| {
+                DbErr::Custom(
+                    "Could not convert raw name bytes into string for comparison".to_owned(),
+                )
+            })?;
+
+            let cond = Condition::all().add(asset_data::Column::RawName.like(name_as_str));
```

Comment:
For partial match, we should do sth like this:
```suggestion
            let name_expr = SimpleExpr::Custom(format!("chain_data->>'name' LIKE '%{}%'", name_as_str).into());
            conditions = conditions.add(name_expr);
```
Postgres is throwing an error here: `RawName.like(name_as_str))` (tested locally just now), because the LIKE operator is meant for textual data, and we're trying to perform a text-based operation on binary data (RawName is stored as bytea).